arangopipe/tests/AQL/Running_AQL.md

```diff
@@ -0,0 +1,81 @@
+# Running AQL queries  on Arangopipe
+
+Apart, from using the **Arangopipe** search feature, it is also possible to query machine learning meta-data using the query feature in **ArangoDB**. An excerpt of the feature is shown in the figure below.
+<figure>
+  <img src="AQL_feature.png" alt="my alt text" width=90%/>
+  <figcaption style ="text-align:center">Figure 1.</figcaption>
+</figure>
+The left pane shows an editor area where we can enter queries. We can the execute the query by clicking on the *_Execute_* button in the lower right hand corner. Let us illustrate this with an example. The dataset used for illustration is the wine dataset used to illustrate how **Arangopipe** can be used with *_mlflow_* (see the *_mlflow_* folder). We will illustrate the following operations:
+
+1. Deleting a dataset from the database
+2. Deleting a featureset from the database
+3. Inserting a dataset into the database
+4. Inserting a featureset into the database
+5. Linking the featureset and the dataset
+
+If you would like to learn more about AQL, please view the [AQL documentation](https://www.arangodb.com/docs/stable/aql/). The [tutorials](https://www.arangodb.com/docs/stable/tutorials.html) section of the documentation may also be of interest.
+
+## Deleting a dataset from the database
```

Comment:
Just curious: why do we show a delete first, IMO that should only be rare occasions as you destroy an audit trail.

Reply:
Doing the delete first makes no assumptions about the state of the database while executing the actions in the sequence listed in the markdown file. If the user had executed this exercise a few times, the inserted results will be in the database. This sequence gaurantees the same results each time and serves to illustrate basic CRUD operations.

Reply:
IMO delete destroys a potential audit trail, so maybe we should even consider whether that needs a special role/permissions from the arangoml UI (just documenting a thought here ;-) )